README.md

```diff
@@ -56,6 +56,12 @@ The root account is derived from the network passphrase and if the network passp
 
 *Note*: The standalone network in this container is not suitable for any production use as it has a fixed root account. Any private network intended for production use would also required a unique network passphrase.
 
+In order to get started quickly, you can deploy the docker image to a DigitalOcean server by clicking the button below. It will create a container in *ephemeral mode* on the *standalone network* that can be connected to. Note: you will need to create a DigitalOcean account if you don't have one.
```

Comment:
I think we should mention about the unique network passphrase, by just stating that the network passphrase in this deployment will be randomly defined.

Reply:
👍  https://github.com/stellar/docker-stellar-core-horizon/pull/276/commits/82fb3526e17c70224d8171f009dbbffc0f9790b5